app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  after_save :update_likes_counter
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  private
+
+  def update_likes_counter
+    post.update(likescounter: post.likes.all.length)
```

Comment:
- [ ] [OPTIONAL] Instead of using the update method, kindly consider using the rails increment method (eg: post.increment!(:likes_counter).  The update method is used to modify attributes of records, while the increment method is specifically used to modify numeric attributes by incrementing their values. Hence the increment method is the appropriate method to be used here. :pray: